.eslintrc

```diff
@@ -76,22 +76,16 @@
     "import/prefer-default-export": "off",
     "import/no-named-as-default": "error",
     "import/no-cycle": "error",
-    "import/no-default-export": "error",
+    "import/no-default-export": "off",
+    "import/no-unresolved": "off",
```

Comment:
Looks like these changes are unrelated to this PR, can you remove changes to the lint file?

Reply:
I included these as I was receiving errors while compiling locally. I will revert them.

Reply:
Hmm odd, with your code or with other code? The current code in the codebase shouldn't have any errors with the lint settings besides maybe in the demo project (which we don't use 😅)

Reply:
Neither. The problem occurs when I try to commit with git. It won't let me unless the lint issues are resolved.

Reply:
I think it's because of the changes to the demo components though, lint staged it linting when you commit and the lint errors are from the demo files

Reply:
[STARTED] Preparing lint-staged...
[SUCCESS] Preparing lint-staged...
[STARTED] Running tasks for staged files...
[STARTED] package.json — 7 files
[STARTED] *.{ts,tsx,js,jsx} — 3 files
[STARTED] eslint --fix
[FAILED] eslint --fix [FAILED]
[FAILED] eslint --fix [FAILED]
[FAILED] eslint --fix [FAILED]
[STARTED] Applying modifications from tasks...
[SKIPPED] Skipped because of errors from tasks.
[STARTED] Reverting to original state because of errors...
[SUCCESS] Reverting to original state because of errors...
[STARTED] Cleaning up temporary files...
[SUCCESS] Cleaning up temporary files...

✖ eslint --fix:

/Users/kamil/Projects/VEClim/recharts/src/cartesian/BrushY.tsx
  308:39  error  Unexpected empty arrow function                          @typescript-eslint/no-empty-function
  310:39  error  Unexpected empty arrow function                          @typescript-eslint/no-empty-function
  543:27  error  'isSlideMoving' is assigned a value but never used       @typescript-eslint/no-unused-vars
  543:42  error  'isTravellerMoving' is assigned a value but never used   @typescript-eslint/no-unused-vars
  543:61  error  'isTravellerFocused' is assigned a value but never used  @typescript-eslint/no-unused-vars

✖ 5 problems (5 errors, 0 warnings)

husky - pre-commit hook exited with code 1 (error)

Reply:
I'll fix these and see if it works

Reply:
Oh yeah those are actual lint errors that need fixed

Reply:
Okay, it committed without any errors and with BrushYDemo intact (might just be my luck). Just to note, the part I included in the storybook does nothing. It is the demo that connects BrushY with YAxis. I am not sure how to have the same functionality in the storybook, but it might not be necessary. Hance the reason to name it as a "dummy" component.